src/scripts/main.js

```diff
@@ -1,3 +1,24 @@
 'use strict';
 
-// write your code here
+function createCopy(table, columnIndex, copyTo) {
+  const column = getColumn(table, columnIndex);
+
+  changeColumn(table, column, copyTo);
+}
+
+function changeColumn(tablet, column, copyTo) {
+  column.forEach((element, i) => {
+    if (copyTo === undefined) {
+      tablet.rows[i].append(element.cloneNode(true));
+    } else {
+      tablet.rows[i].cells[copyTo].innerHTML = element.innerHTML;
+    }
+  });
+}
+
+function getColumn(tablet, column) {
```

Comment:
The parameter `tablet` should be renamed to `table` for consistency with the `createCopy` function.